gui-backend/src/main/java/com/rln/gui/backend/implementation/balanceManagement/data/AccountInfo.java

```diff
@@ -12,12 +12,14 @@
 @AllArgsConstructor
 public class AccountInfo {
     private String provider;
+    private String owner;
     private String iban;
     private String assetCode;
 
   public AccountInfo(Balance balance) {
     provider = balance.provider;
     iban = balance.iban;
     assetCode = balance.currency;
+    owner = balance.owner.party.orElse(balance.owner.name);
```

Comment:
I suggest adding `Objects.requireNonNull(balance.owner)` before this.

Reply:
Added. There might be other places where we are missing such a call.